README.md

```diff
@@ -123,6 +123,8 @@ Darling's features are built to work well for real-world projects.
 -   **Span access**: Use `darling::util::SpannedValue` in a struct to get access to that meta item's source code span. This can be used to emit warnings that point at a specific field from your proc macro. In addition, you can use `darling::Error::write_errors` to automatically get precise error location details in most cases.
 -   **"Did you mean" suggestions**: Compile errors from derived darling trait impls include suggestions for misspelled fields.
 -   **Struct flattening**: Use `#[darling(flatten)]` to remove one level of structure when presenting your meta item to users. Fields that are not known to the parent struct will be forwarded to the `flatten` field.
+-   **Custom shorthand**: Use `#[darling(from_word = ...)]` on a struct or enum to override how a simple word is interpreted. By default, it is an error for your macro's user to fail to specify the fields of your struct, but with this you can choose to instead produce a set of default values. This takes either a path or a closure whose signature matches `FromMeta::from_word`.
```

Comment:
I considered not requiring the outer `Result`, as that would greatly increase the likelihood of path values "just working", and given that there are no inputs to this function it seems unlikely that someone would want to return an `Err`.

Unfortunately, I feel like this sort of common-path optimization has burned `darling` in the past, either in the form of increased complexity or in making certain concepts impossible to express; maybe the `from_word` function is calling a method on a type parameter and that can sometimes fail.

Given the signatures for these two methods, I expect the closure form will be more common here to handle the `Ok` and `Some` wrapping.